### PR TITLE
fix: Outdated information about Xidian University

### LINKED ISSUE #557: 对西安电子科技大学的军训情况有误
### MtF.wiki URL

https://mtf.wiki/zh-cn/docs/campus/china/xdu/

### 内容

本人为了西安电子科技大学2020级本科生，刚刚参加完军训，发现网站上所写完全背离事实。
Screenshot_2022-07-21-16-21-58-37_40deb401b9ffe8e1df2f1cc5ba480b12

但实际上，西电军训毫无头发要求。长头发生理男生（多为顺性别男性)均扎辫子参加军训。
此外，根据我对2019级学长的调查，他们的军训亦没有头发长度要求。
附本人军训时照片为证
IMG_20220627_233907

### PR DESCRIPTION
**Pull request by CamberLoid**
Fix #557

### CODE REVIEW

**Comment by CamberLoid** [comment]
Request review @RebelRedGuards

**Comment by CamberLoid** [comment]
Request review @RebelRedGuards @septs

File: content/zh-cn/docs/campus/china/XDU.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ title: 西安电子科技大学
 
 ### 体育与军训
 
-军训方面，可以开医学证明申请免训，已有同伴因为 HRT 成功申请免训。（需注明不适合体育运动/不适合军训等，目前要求三甲医院）若参加则无留头发可能。
+军训方面，可以开医学证明申请免训，已有同伴因为 HRT 成功申请免训。（需注明不适合体育运动/不适合军训等，目前要求三甲医院）。另外，最近的军训对头发已不作要求。
```
Comment (septs) [inline content/zh-cn/docs/campus/china/XDU.md:19]: 不应使用相对时间表述

Comment (RebelRedGuards) [reply content/zh-cn/docs/campus/china/XDU.md:19]: 可以写“至少从19级开始，军训对头发无要求”

Comment (CamberLoid) [reply content/zh-cn/docs/campus/china/XDU.md]: Fixed. Request review.

**Pull request merged**